hedera-node/src/main/java/com/hedera/services/state/merkle/internals/CopyOnWriteIds.java

```diff
@@ -172,9 +172,9 @@ public void setNativeIds(long[] ids) {
 		return ids;
 	}
 
-	public List<TokenID> getAsIds() {
+	public List<TokenID> getAsIds(int count) {
```

Comment:
Add javadoc what this count input is. It's not clear without looking at the code and may lead to confusion in the future. Prefer renaming the variable as maxSize.

Reply:
Actually we no longer need this. Removing it